### PR TITLE
add snapshotter client

### PR DESCRIPTION
**Pull request by dgnorton**
- [ ] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: services/snapshotter/client.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	}
+	i := 8
+
+	// Size of the meta store bytes.
+	length := int(binary.BigEndian.Uint64(b[i : i+8]))
+	i += 8
+	metaBytes := b[i : i+length]
+	i += int(length)
+
+	// Size of the node.json bytes.
```
Comment (e-dard) [inline services/snapshotter/client.go]: Is this cruft supposed to be in here still?

**Comment by e-dard** [comment]
Apart from small comment LGTM

File: services/snapshotter/client.go (hunk 1 above)
Comment (jwilder) [inline services/snapshotter/client.go]: Changing that may break the backup format.

Comment (dgnorton) [inline services/snapshotter/client.go]: @jwilder not sure I understand.  This client is only used by by the clustering code.

Comment (jwilder) [inline services/snapshotter/client.go]: Ah.  I missed that this was the client.  I thought I was looking at the snapshotter service.

**Comment by jwilder** [comment]
I think you can remove the commented code.  :+1:  otherwise.

**Pull request merged**